### PR TITLE
Heat-in utility

### PR DESCRIPTION
**Pull request by dmwinslow**
This adds a python equivalent to the `heatin` utility, along with tests that assert that the output files are identical to those generated by the matlab versions for a number of model runs.

I consider this the most basic interface possible (called from the command line as described in the `README.md` changes), and I intend to improve on it in a future PR by supporting automatic detection of files like we have in the Matlab version. This is also missing any kind of automatic plotting - again that needs to be added separately.

### CODE REVIEW

**Review by hydrocrust — APPROVED** [review]
Ok, I think I get this. BTW, the plot has proven helpful to make sure that correct values were used. Another way to address that could be to give high, low, and median values of HF. The input is a total heating rate, becomes HF when dividing by area, so the units are hard to parse. Also, the units are MJ I think...

**Comment by dmwinslow** [comment]
> Ok, I think I get this. BTW, the plot has proven helpful to make sure that correct values were used. Another way to address that could be to give high, low, and median values of HF. The input is a total heating rate, becomes HF when dividing by area, so the units are hard to parse. Also, the units are MJ I think...Also, I like the plots, but this causes an issue with 2D versus 3D, I acknowledge that. Could be a part of the code that checks to see if X or Y are constant to tell if it is 2D or 3D domain...

Yep, I've already started working on the plots separately - they'll definitely be included. The plan is to check for variance in X/Y as you suggest and show 2D as line plots, 3D as contour plots. Hopefully I'll have something to show you by our next meeting.
